src/returnError.js

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable max-len */
+const errors = {
+  textmissing: 'Text to convert is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+  caseMissing: '"toCase" query param is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+  notSupportedCase: 'This case is not supported. Available cases: SNAKE, KEBAB, CAMEL, PASCAL, UPPER.',
+};
+
+const supportedCases = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+
+const returnError = (text, toCase) => {
```

Comment:
check function naming
when you call this function you want to check and set correct errors, not return something from your code